article-introduction_into_diffusion_and_latent_diffusion_models/article.md

```diff
@@ -0,0 +1,88 @@
+# Introduction into Diffusion & Latent Diffusion Models
+
+![Figure 1. Latent Diffusion Model Architecture](src/images/The-architecture-of-the-latent-diffusion-model-LDM-that-is-considered-a-revolutionary.png)
+
+_Figure 1. Latent Diffusion Model Architecture_
+
+# Introduction
+
+In the last couple of years, large text-to-image models have become more and more powerful, achieving state-of-the-art results. These advancements have sparked interest in the domain and given birth to multiple commercial projects offering text-to-image generation on subscription or token-based models. Although used daily, their users rarely understand the way they work. So, in this article, I will explain the work of the Stable Diffusion model, one of the most popular text-to-image models to date.
+
+As suggested by its name, Stable Diffusion is a type of diffusion model called a Latent Diffusion Model. It was first described in [**"High-Resolution Image Synthesis with Latent Diffusion Models"** by **Robin Rombach, Andreas Blattmann, Dominik Lorenz, Patrick Esser, and Björn Ommer**](https://arxiv.org/abs/2112.10752). At its core, there are two layers: the convolutional layer, which is responsible for image generation, and the self-attention layer, which is responsible for text processing.
```

Comment:
Related to your last sentence In Stable Diffusion, text processing is handled by a separate text encoder (often a transformer-based model like CLIP's text encoder), not by self-attention layers within the convolutional neural network (CNN). The self-attention layers within the U-Net are used to capture long-range dependencies in the latent image representations, not to process text.